cmd/dispatcher/main.go

```diff
@@ -37,6 +38,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.namespace, "namespace", "", "Namespace where the calculations exists")
 	fs.StringVar(&o.redisURL, "redis-url", "", "Redis database url host")
 	fs.StringVar(&o.redisPasswordFile, "redis-password-file", "", "Path of the Redis database password file")
+	fs.StringVar(&o.redisSortedSetName, "redis-sorted-set-name", "", "Name of the Redis sorted set")
```

Comment:
One last thing. We should default it to `vz` to keep backward compatibility otherwise the deployment will start failing.